__tests__/app.test.js

```diff
@@ -0,0 +1,42 @@
+const request = require("supertest");
+const app = require("../app");
+const data = require("../db/data/test-data/");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const index = require('../db/data/test-data/index')
+
+
+beforeEach(() => {
+    return seed(data);
+});
+
+afterAll(() => {
+    db.end();
+});
+
+
+describe('GET /api/topics', () => {
+    test('GET 200 /api/topics', () => {
+        return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({ body: { topics } }) => {
+                expect(topics).toEqual(index.topicData)
```

Comment:
I wouldnt do it this way, its bad practise. Better to check the length, do a forEach then make sure it contains the keys we want:)